app/models/specialist_document.rb

```diff
@@ -86,7 +86,11 @@ def update(params)
   def published?
     editions.any?(&:published?)
   end
-  alias_method :has_ever_been_published?, :published?
+
+  def has_ever_been_published?
+    return false if @editions.size == 1 && needs_exporting?
+    published?
```

Comment:
When you create a new draft it is marked as needing exporting (it's exported_at date hasn't been set).  When you publish that document the first thing that happens is the state is changed from draft to published, then we write out a final draft to the publishing-api; at which point if we ask if the document has ever been published to work out some logic about how to represent the document for exporting to publishing-api.  Without this change it'll say it has, when in fact it's in the middle of being published and so we were seeing the wrong content in drafts being sent to the publishing-api.  It's messy as all heck, yes, but that's par for the course for this app.